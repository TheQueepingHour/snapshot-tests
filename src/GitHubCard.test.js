import { render, screen } from '@testing-library/react'
import myCard from './GitHubCard'
import renderer from 'react-test-renderer'

test('renders my card', () => {
    const card = renderer.create(<myCard />).toJSON()
    expect(card).toMatchSnapshot()
})