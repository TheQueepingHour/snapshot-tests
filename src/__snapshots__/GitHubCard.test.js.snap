// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders my card 1`] = `<myCard />`;
